ui/components-react/src/components-react/filter-builder/FilterBuilder.scss

```diff
@@ -0,0 +1,132 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+.fb-component {
+  --fb-drag-handle-width: 1.25rem;
+  --fb-operator-width: 4.5rem;
+  --divider-height: var(--iui-size-s);
+  --divider-width: var(--iui-size-2xs);
+  --divider-background-color: var(--iui-color-border-subtle);
+
+  // set the CSS var for the width of the logical operator (if size=small)
+  [data-iui-size="small"] {
+    --fb-operator-width: 4rem;
+  }
+
+  [data-iui-size="large"] {
+    --fb-operator-width: 5rem;
+    --fb-drag-handle-width: 1.5rem;
+  }
+
+  // set the width of the drag handle
+  .fb-row-drag-handle {
+    width: var(--fb-drag-handle-width);
+  }
+
+  // set the width of the logical operator
+  .fb-row-operator {
+    width: var(--fb-operator-width);
+  }
+
+  // indent "child" components
+  .fb-component-indent {
+    margin-left: calc(var(--level) * var(--fb-operator-width));
+
+    > div {
+      border-top: none; // collapse border with previous sibling
+    }
+
+    &.fb-component-draggable {
+      margin-left: calc(
+        var(--level) * (var(--fb-drag-handle-width) + var(--fb-operator-width))
+      );
+    }
+  }
+
+  .fb-component-row:not(.fb-component-indent) {
+    ~ .fb-component-indent {
+      > div {
+        border-radius: 0;
+      }
+
+      &:nth-last-child(1 of .fb-component-indent) > div {
```

Comment:
@FlyersPh9 are we fine using css that is not 1 year old yet for most browsers ? Are these expected to be of different classes than `.fb-component-indent` ?